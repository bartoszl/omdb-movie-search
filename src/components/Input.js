import styled from 'styled-components';

const Input = styled.input`
  padding: 0.5rem 0.25rem;
  border-radius: 3px;
  border: none;
`;

export default Input;