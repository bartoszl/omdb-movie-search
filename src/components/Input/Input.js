import styled from 'styled-components';

const Input = styled.input`
  padding: 0.75rem 0.25rem;
  border-top-left-radius: 3px;
  border-bottom-left-radius: 3px;
  border: none;
  width: 16rem;
`;

export default Input;
