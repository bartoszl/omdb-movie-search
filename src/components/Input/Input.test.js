import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Input from './Input';

describe('Input', () => {
  afterEach(() => {
    cleanup();
  });

  test('matches snapshot', () => {
    const { asFragment } = render((
      <Input name="test" placholder="Search..." />
    ));

    expect(asFragment()).toMatchSnapshot();
  });
});
