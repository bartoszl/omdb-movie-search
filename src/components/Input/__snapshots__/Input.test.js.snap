// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input matches snapshot 1`] = `
<DocumentFragment>
  <input
    class="sc-bdnylx cUPCfY"
    name="test"
  />
</DocumentFragment>
`;
